apps/explorer/src/components/activity/Activity.tsx

```diff
@@ -96,52 +86,19 @@ export function Activity({ initialLimit, disablePagination }: ActivityProps): JS
-                                        <FilterList className="h-md w-md text-neutral-10 dark:text-neutral-92" />
+                    {pollingTxnTableEnabled &&
+                        selectedCategory === ActivityCategory.Transactions && (
+                            <div>
+                                <Toggle
+                                    label="Show System Transactions"
+                                    isToggled={showSystemTransactions}
+                                    size={ToggleSize.Small}
+                                    onChange={() =>
+                                        setshowSystemTransactions(!showSystemTransactions)
                                     }
                                 />
+                                <PlayPause paused={paused} onChange={handlePauseChange} />
                             </div>
-                            <div className="absolute bottom-0 right-0 z-10 translate-y-full">
-                                <Transition
-                                    show={showTransactionDropdown}
-                                    enter="transition duration-300"
-                                    enterFrom="opacity-0 scale-75"
-                                    enterTo="opacity-100 scale-100"
-                                    leave="transition duration-150"
-                                    leaveFrom="opacity-100 scale-100"
-                                    leaveTo="opacity-0 scale-75"
-                                >
-                                    <Dropdown>
-                                        <ListItem
-                                            hideBottomBorder
-                                            onClick={() =>
-                                                setShowSystemTransaction(!showSystemTransactions)
-                                            }
-                                        >
-                                            <div className="flex flex-row gap-x-xs">
-                                                <span className="w-max text-label-lg">
-                                                    Show System Transactions
-                                                </span>
-                                                <Checkbox isChecked={showSystemTransactions} />
-                                            </div>
-                                        </ListItem>
-                                    </Dropdown>
-                                </Transition>
-                            </div>
-                        </>
-                    ) : null*/}
-                    {/* todo: re-enable this when rpc is stable */}
-                    {pollingTxnTableEnabled &&
-                        selectedCategory === ActivityCategory.Transactions && (
-                            <PlayPause paused={paused} onChange={handlePauseChange} />
                         )}
                 </div>
```

Comment:
even if it was like that already, would you mind including the div in the condition? :pray:  it’ll help prevent an empty div
```suggestion
                {pollingTxnTableEnabled && selectedCategory === ActivityCategory.Transactions && (
                    <div className="absolute inset-y-0 -top-1 right-sm flex items-center gap-sm text-2xl">
                        <div>
                            <Toggle
                                label="Show System Transactions"
                                isToggled={showSystemTransactions}
                                size={ToggleSize.Small}
                                onChange={() => setshowSystemTransactions(!showSystemTransactions)}
                            />
                            <PlayPause paused={paused} onChange={handlePauseChange} />
                        </div>
                    </div>
                )}
```

Reply:
thank you! I changed it